agent/agentbootstrap/bootstrap.go

```diff
@@ -268,7 +269,7 @@ func (b *AgentBootstrap) Initialize(ctx context.Context) (_ *state.Controller, r
 		modeldefaultsbootstrap.SetCloudDefaults(stateParams.ControllerCloud.Name, stateParams.ControllerInheritedConfig),
 		secretbackendbootstrap.CreateDefaultBackends(model.ModelType(modelType)),
 		controllerModelCreateFunc,
-		modelbootstrap.CreateLocalModelRecord(controllerModelUUID, controllerUUID, agentVersion),
+		modelbootstrap.CreateLocalModelRecord(controllerModelUUID, controllerUUID, agentVersion, agentbinary.AgentStreamReleased),
```

Comment:
To confirm - here and elsewhere use "released" for now and we'll parameterise later. 
I think TODOs would be good.

Reply:
This task requires this to be done so I can't finish the task off until it is. Next PR does it all.